src/test/java/uk/nhs/ctp/controllers/CdssControllerComponentTest.java

```diff
@@ -66,6 +69,9 @@
   @Autowired
   private CdssSupplierRepository cdssRepository;
 
+  @Autowired
+  private AuditService auditService;
```

Comment:
It could be a little cleaner to wire in the thread store and just add a blank session in a setup

Reply:
Done